libservice/src/IpAddressNetlinkChecker.cpp

```diff
@@ -131,11 +139,73 @@ static std::optional<IPv4int> getMappedIPv4Addr(const in6_addr& addr) {
 	return ipv4Binary;
 }
 
+IpAddressNetlinkChecker::ipv6Range IpAddressNetlinkChecker::parseIpv6Range(const std::string& range) const {
```

Comment:
is this parsing necessary? the method from ipv4 where are defined constexpr addresses and masks seems easier?.

Reply:
Ok, maybe it's not a harm to leave it.